### ISSUE TITLE
Error in readRDS("cloudml/deploy.rds")

### ISSUE THREAD

**Issue by KevinDukelow5** [rstudio/cloudml#185]
Running into an issue deploying any training runs from our Rstudio Server. The server is running in a container in a GCP VM (a Rocker image). The relevant log messages are:

-- Error in   readRDS("cloudml/deploy.rds") :
-- cannot read workspace version 3 written by   R 3.6.0; need R 3.5.0 or newer
-- Execution halted
-- Command ['Rscript', '/root/.local/lib/python2.7/site-packages/cloudml-model/cloudml/deploy.R'] failed: exit code 1
-- Command '['python', '-m', u'cloudml-model.cloudml.deploy', u'Rscript', '--job-dir', u'gs://supplychain-sandbox/r-cloudml/staging']' returned non-zero exit status 1.
-- Module completed; cleaning up.

Have gone through a number of attempts to fix but running out of options.

[ml_job_job_id_cloudml_2019_03_10_041202739__logs__2019-03-09T11-16.txt](https://github.com/rstudio/cloudml/files/2949071/ml_job_job_id_cloudml_2019_03_10_041202739__logs__2019-03-09T11-16.txt)

**Comment by Sophia0616** [comment]
From the R documentation from 3.6, the rds written by R 3.6 is not readable by previous versions. I was able to fix this:

Sys.setenv(R_CONFIG_ACTIVE = "cloudml", R_DEFAULT_SAVE_VERSION = 2, R_DEFAULT_SERIALIZE_VERSION = 2 )

placed it in deploy.R

You can use my repo https://github.com/Sophia0616/cloudml or fork this and make the changes yourself.

I have no experience in writing cran libraries of any kind, so I hope someone with more experience than I do can tell me if I did something stupid. Always happy to learn.

Hope that helps :)

**Comment by kevinushey** [comment]
What version of cloudml are you using? I believe this should've been fixed a while back:

https://github.com/rstudio/cloudml/commit/fe081744b4f0d3f156b51853b9cbd2b2cb809402#diff-308c00e39a4798634489761b6636ded1

**Comment by RoelPi** [comment]
> What version of cloudml are you using? I believe this should've been fixed a while back:
> 
> [fe08174#diff-308c00e39a4798634489761b6636ded1](https://github.com/rstudio/cloudml/commit/fe081744b4f0d3f156b51853b9cbd2b2cb809402#diff-308c00e39a4798634489761b6636ded1)

Hi there

@kevinushey: I am using the latest version of the cloudml package (0.6.0) and I run into the same issue. 

I have done @Sophia0616 's solution but it doesn't work. I checked the zip file in the staging directory on google storage, where the jobs are saved and the proposed change is definitely in there.

I've spent two nights trying to solve this but still no avail. Hoping some of you might be able to help out.

Best regards
Roel
